project_department/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import project
```

Comment:
There was a discussion on that, and they are not required on `__init__.py` files